shell/platform/embedder/embedder.h

```diff
@@ -1326,6 +1326,12 @@ typedef void (*FlutterUpdateSemanticsCallback2)(
     const FlutterSemanticsUpdate2* /* semantics update */,
     void* /* user data*/);
 
+typedef void (*FlutterChannelListenedToCallback)(
+    const char* data /* name */,
+    size_t data_len /* name.size */,
+    bool /* listening */,
+    void* /* user data */);
```

Comment:
I would use the info struct pattern. For example: https://github.com/flutter/engine/blob/83fdadaf12f00ef0ca3aa9401918a2486fcb89a2/shell/platform/embedder/embedder.h#L486-L503

This allows us to add new arguments to the callback if we ever need to in the future.